### PR TITLE
Update to latest liquid fire and paint.

### PR DESCRIPTION
**Pull request by willrax**
:8ball:

### CODE REVIEW

File: blueprints/ember-cli-paint/index.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ module.exports = {
...
     return this.addPackagesToProject([
-      { name: 'liquid-fire', target: '0.17.1' },
+      { name: 'liquid-fire', target: '0.19.0' },
       { name: 'ember-rl-dropdown', target: 'git+https://git@github.com/alphasights/ember-rl-dropdown.git' },
       { name: 'ember-cli-tooltipster', target: '0.0.6' },
       { name: 'broccoli-sass', target: '0.4.0' }
     ]).then(function() {
       return this.addBowerPackagesToProject([
-        { name: 'paint', target: '0.6.10' },
+        { name: 'paint', target: '0.7.20' },
```
Comment (deioo) [inline blueprints/ember-cli-paint/index.js]: It's 0.7.2, hopefully we'll reach 0.7.20 by the end of the week :)

**Comment by matteodepalo** [comment]
:+1:

File: blueprints/ember-cli-paint/index.js [hunk 2]
```diff
@@ -5,13 +5,13 @@ module.exports = {
...
     return this.addPackagesToProject([
-      { name: 'liquid-fire', target: '0.17.1' },
+      { name: 'liquid-fire', target: '0.19.0' },
       { name: 'ember-rl-dropdown', target: 'git+https://git@github.com/alphasights/ember-rl-dropdown.git' },
       { name: 'ember-cli-tooltipster', target: '0.0.6' },
       { name: 'broccoli-sass', target: '0.4.0' }
     ]).then(function() {
       return this.addBowerPackagesToProject([
-        { name: 'paint', target: '0.6.10' },
+        { name: 'paint', target: '0.7.2' },
```
Comment (duboff) [inline blueprints/ember-cli-paint/index.js]: There is actually paint 0.7.5 now

**Comment by deioo** [comment]
I think this is deprecated @willrax ?

**Comment by eugeniodepalo** [comment]
Should we close this?

**Comment by willrax** [comment]
done

**Comment by duboff** [comment]
:+1: :smile_cat:

**Comment by willrax** [comment]
@duboff :wave:

**Pull request closed without merging**